### PR TITLE
docs: brush up writing tests section

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: docs/src/writing-tests-js.md [hunk 1]
```diff
@@ -3,90 +3,147 @@ id: writing-tests
 title: "Writing Tests"
 ---
 
-Playwright assertions are created specifically for the dynamic web. Checks are automatically retried until the necessary conditions are met. Playwright comes with [auto-wait](./actionability.md) built in meaning it waits for elements to be actionable prior to performing actions. Playwright provides a [test](./api/class-test.md) function to declare tests and the [expect](https://jestjs.io/docs/expect) function to write assertions.
+Playwright tests are simple, they
```
Comment (mxschmitt) [inline docs/src/writing-tests-js.md:6]: Make the changes in the other writing-tests* files as well?

File: docs/src/writing-tests-js.md [hunk 2]
```diff
@@ -3,90 +3,147 @@ id: writing-tests
...
-### Assertions
+```js
+await page.getByRole('link', { name: 'Get started' }).click();
+```
+
+
+### Basic actions
+
+This is the list of the most popular Playwright actions. Note that there are
+many more, so make sure to check the [Page API](./api/class-page.md) section to
```
Comment (mxschmitt) [inline docs/src/writing-tests-js.md:101]: Locator API?

File: docs/src/writing-tests-js.md [hunk 3]
```diff
@@ -3,90 +3,147 @@ id: writing-tests
...
-### Locators
 
-[Locators](./locators.md) are the central piece of Playwright's auto-waiting and retry-ability. Locators represent a way to find element(s) on the page at any moment and are used to perform actions on elements such as `.click` `.fill` etc.
+Here is the list of the most popular async assertions. Note that there are [many more](./test-assertions.md) to get familiar with:
 
-```js
-const getStarted = page.getByRole('link', { name: 'Get started' });
+| Assertion | Description |
+| :- | :- |
+| [expect(locator).toBeChecked](./test-assertions#locator-assertions-to-be-checked) | Checkbox is checked |
```
Comment (mxschmitt) [inline docs/src/writing-tests-js.md:135]: Why not use method: …?

**Review by dgozman — APPROVED** [review]

File: docs/src/writing-tests-js.md (hunk 1 above)
Comment (pavelfeldman) [reply docs/src/writing-tests-js.md:6]: One step at a time!

File: docs/src/writing-tests-js.md (hunk 2 above)
Comment (pavelfeldman) [reply docs/src/writing-tests-js.md:101]: Yup!

File: docs/src/writing-tests-js.md (hunk 3 above)
Comment (pavelfeldman) [reply docs/src/writing-tests-js.md:135]: Because of no good reason apparently - I mistyped it, it did not work, so I worked around. It does work!

**Pull request merged**